tests/retry_imports_test.go

```diff
@@ -0,0 +1,66 @@
+package tests_test
+
+import (
+	"flag"
+	"strings"
+	"time"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	tests "kubevirt.io/kubevirt-ansible/tests/framework"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+const (
+	invalidURL    = "https://noneexist.com"
+	dvName        = "golden-dv"
+	dstDVFilePath = "/tmp/test-dv.json"
+	rawDVFilePath = "tests/manifests/golden-dv.yml"
+)
+
+var _ = FDescribe("Importing image using CDI with invalid URL", func() {
+
+	flag.Parse()
+
+	ktests.BeforeAll(func() {
+		ktests.BeforeTestCleanup()
+		By("Create a DV with invalid image URL")
+		tests.ProcessTemplateWithParameters(rawDVFilePath, dstDVFilePath, "DV_NAME="+dvName, "EP_URL="+invalidURL)
+		tests.CreateResourceWithFilePathTestNamespace(dstDVFilePath)
+	})
+
+	Context("DV with invalid image url", func() {
+		It("Importer pod should be in CrashLoopBackOff status", func() {
```

Comment:
Use By, It doesn't guarantee the run order.

Reply:
Done